code/game/objects/structures/tank_holder.dm

```diff
@@ -35,13 +35,24 @@
 
 /obj/structure/tank_holder/examine(mob/user)
 	. = ..()
+	. += "It is [anchored ? "wrenched to the floor." : "unsecured. A wrench should fix that."]<br/>"
```

Comment:
normally when you do this kind of examine message for constructables you want to make the description a little more subtly hint at the holes for **bolts** at the base rather than bluntly going APPLY WRENCH HERE

Reply:
It was a copy from other files but sure.